CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 librdkafka v2.11.0 is a feature release:
 
+* [KIP-1102](https://cwiki.apache.org/confluence/display/KAFKA/KIP-1102%3A+Enable+clients+to+rebootstrap+based+on+timeout+or+error+code) Enable clients to rebootstrap based on timeout or error code (#4981).
```

Comment:
We should add which error code as well. `RD_KAFKA_RESP_ERR_REBOOTSTRAP_REQUIRED`

Reply:
this is just the KIP title it's all described there